tests/test_deterministic_signals.py

```diff
@@ -177,16 +177,6 @@ def test_physical_ephem_model(self):
             msg = "Ephemeris delay does not match pre-computed values"
             assert np.allclose(d1, eph_wf, rtol=1e-10), msg
 
-        # test PINT exception raising
-        elif isinstance(self.psr, enterprise.pulsar.PintPulsar):
-            with self.assertRaises(NotImplementedError) as context:
-                eph1 = deterministic_signals.PhysicalEphemerisSignal(sat_orb_elements=True)
-                e1 = eph1(self.psr)
-
-                msg = "Physical Ephemeris model is not compatible with PINT "
-                msg += "at this time."
-                self.assertEqual(msg, str(context.exception))
-
 
 class TestDeterministicSignalsPint(TestDeterministicSignals):
```

Comment:
Yeah, oops. I missed that the subclassing took care of all of these tests. I did add a test for `setIII` here as well, since that is now our default. That did not include the saved vector check though.